src/graphql/data/TopTokens.ts

```diff
@@ -8,18 +8,19 @@ import {
   sortMethodAtom,
 } from 'components/Tokens/state'
 import { useAtomValue } from 'jotai/utils'
-import { useCallback, useEffect, useMemo, useState } from 'react'
+import { useCallback, useLayoutEffect, useMemo, useState } from 'react'
 import { fetchQuery, useLazyLoadQuery, useRelayEnvironment } from 'react-relay'
 
-import { ContractInput, TopTokens_TokensQuery } from './__generated__/TopTokens_TokensQuery.graphql'
+import { ContractInput, HistoryDuration, TopTokens_TokensQuery } from './__generated__/TopTokens_TokensQuery.graphql'
 import type { TopTokens100Query } from './__generated__/TopTokens100Query.graphql'
-import { toHistoryDuration, useCurrentChainName } from './util'
+import { toHistoryDuration } from './util'
+import { useCurrentChainName } from './util'
 
-export function usePrefetchTopTokens() {
-  const duration = toHistoryDuration(useAtomValue(filterTimeAtom))
+export function usePrefetchTopTokens(duration: HistoryDuration) {
   const chain = useCurrentChainName()
-  const args = useMemo(() => ({ chain, duration }), [chain, duration])
-  return useLazyLoadQuery<TopTokens100Query>(topTokens100Query, args)
+
+  const top100 = useLazyLoadQuery<TopTokens100Query>(topTokens100Query, { duration, chain })
```

Comment:
why not just return this value instead of assigning and returning that?

Reply:
that is a good idea haha, fixed.